apps/trustlab/src/payload/access/abilities.js

```diff
@@ -0,0 +1,60 @@
+import { checkRole } from "./checkRole";
+import { ROLE_ADMIN, ROLE_AUTHOR, ROLE_EDITOR } from "./roles";
+
+export const canManageContent = (user) => {
+  // Admin and editors can manage any content
+  if (checkRole([ROLE_ADMIN, ROLE_EDITOR], user)) {
+    return true;
+  }
+
+  // Everyone else can only manage their own content
+  return {
+    createdBy: {
+      equals: user.id,
+    },
+  };
```

Comment:
If we're following [WordPress](https://wordpress.com/support/invite-people/user-roles/#list-of-user-roles) or [Ghost](https://ghost.org/docs/staff/#roles--permissions) model, authors should only be able to manage posts. If I'm reading this correctly, authors will be able to manage pages if/when they create one.

Reply:
@kilemensi We have a different function for pages [`canManagePages`](https://github.com/CodeForAfrica/ui/blob/e27bdf993f740acc66b8767bc4308d15f1657391/apps/trustlab/src/payload/access/abilities.js#L21-L22)

Reply:
... cool, content == posts + media.